apps/src/p5lab/spritelab/CoreLibrary.js

```diff
@@ -11,7 +12,7 @@ import {
 import msg from '@cdo/locale';
 
 export default class CoreLibrary {
-  constructor(p5) {
+  constructor(p5, jsInterpreter) {
```

Comment:
This is where it feels like there must be a better way to do it. I passed the `jsInterpreter` instance here to call `evaluateWatchExpression` below.

Reply:
I don't have a better idea of how to accomplish this, so I think it works for now.

Reply:
Agree that I think this works for now